### PR TITLE
Fix app error messages

### PR DESCRIPTION
**Pull request by der-test**
#### Summary
Fixing inconsistencies of the app_error messages where some messages ended with a period and some didn't. Now all messages end with a period.
The benefit from this change is getting the same user experience all over the system. 

We agreed on this change on the community server in https://community.mattermost.com/core/channels/localization a while ago and I just 
came around finally doing it. @jasonblais 

#### Ticket Link
N/A

#### Checklist
- [x] Includes text changes and localization file (.../i18n/en.json) updates

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -2512,55 +2512,55 @@
...
   },
   {
     "id": "api.user.ldap_to_email.not_available.app_error",
-    "translation": "AD/LDAP not available on this server"
+    "translation": "AD/LDAP not available on this server."
   },
   {
     "id": "api.user.ldap_to_email.not_ldap_account.app_error",
-    "translation": "This user account does not use AD/LDAP"
+    "translation": "This user account does not use AD/LDAP.."
```
Comment (jasonblais) [inline i18n/en.json]:
```suggestion
    "translation": "This user account does not use AD/LDAP."
```

File: i18n/en.json [hunk 2]
```diff
@@ -4384,275 +4384,275 @@
...
   },
   {
     "id": "model.access.is_valid.user_id.app_error",
-    "translation": "Invalid user id"
+    "translation": "Invalid user id."
   },
   {
     "id": "model.authorize.is_valid.auth_code.app_error",
-    "translation": "Invalid authorization code"
+    "translation": "Invalid authorization code".
```
Comment (jasonblais) [inline i18n/en.json]:
```suggestion
    "translation": "Invalid authorization code."
```

File: i18n/en.json [hunk 3]
```diff
@@ -5328,159 +5328,159 @@
...
   },
   {
     "id": "model.post.is_valid.original_id.app_error",
-    "translation": "Invalid original id"
+    "translation": "Invalid original id."
   },
   {
     "id": "model.post.is_valid.parent_id.app_error",
-    "translation": "Invalid parent id"
+    "translation": "Invalid parent id.."
```
Comment (jasonblais) [inline i18n/en.json]:
```suggestion
    "translation": "Invalid parent id."
```

**Comment by jasonblais** [comment]
@der-test quick reminder for the above feedback, and @jespino / @mgdelacroix on the above question :)

**Comment by der-test** [comment]
@jasonblais Thank you for the reminder, approving the changes right now.

**Comment by jasonblais** [comment]
@mgdelacroix I believe some of these strings in en.json will also need to be updated in the respective source and unit test files. Hence, why the build tests are failing.

Would you know if there's a quick way to identify which those strings are? Tim added a period to 874 strings, so I wouldn't want to him/us to manually check each one..

**Comment by mgdelacroix** [comment]
@jasonblais about the automatic check, @jespino is the expert here, but I think it should be possible to add a `govet` check to avoid this being reintroduced in the future.

As per replacing those in the source, I don't think we have the concept of a default error string as we do have in the frontend, or is there something else I'm missing? Either way, one option would be to create a script that runs some `sed` commands that would make the same replacements that we have on the PR but in the source files.

**Comment by jasonblais** [comment]
@mgdelacroix 

> As per replacing those in the source, I don't think we have the concept of a default error string as we do have in the frontend, or is there something else I'm missing? Either way, one option would be to create a script that runs some sed commands that would make the same replacements that we have on the PR but in the source files.

I see, you're probably correct. Would you be able to help take a look why the tests then fail in this case? I wasn't sure from looking at the error logs.

**Comment by mgdelacroix** [comment]
I've sent a patch solving the tests that the CI was complaining about

**Pull request merged**